cpu/cc2538/include/periph_cpu.h

```diff
@@ -290,6 +290,11 @@ typedef struct {
     uint_fast8_t cfg;   /**< timer config word */
 } timer_conf_t;
 
+/**
+ * @brief   Number of available timer channels
+ */
+#define TIMER_CHANNEL_NUMOF     (2U)
```

Comment:
If I understand the implementation of the timer correctly, there are:

- 2 channels, if timer is used in 16 bit mode
- 1 channel, if timer is used in 32 bit mode

Maybe it is better to define `TIMER_CHANNEL_NUMOF` to 1 here and document that a second channel can be "unlocked" by switching to 16 bit mode?